nss-day-cohort-30/bangazon-api-ambitious-albatrosses
# Cb make customer and payment type models

Fixes:
- Bangazon did not have models for Payment Type and Order, but the database has tables for those objects. 

Changes proposed in this request & reasons behind organizational or architectural decisions:
- Without these models Bangazon employees will have a far more difficult time storing and manipulating Payment Type and Order data after it is retrieved from the database. 

Steps to test:
- in Program.cs 

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-In the Main method of Program.cs:
-add "using BangazonAPI.Models"
- insert: 
        PaymentType newPaymentType = new PaymentType()
        {
            AccountNumber = 1234,
            Name = "test account",
            CustomerId = 1
        };
- insert: 
        Order newOrder = new Order()
        {
            CustomerId = 1
        };


Commit message:
- "models for Order and Payment Type created"

Link to feature ticket:
-N/A

@ambitions-albatrosses
